### ISSUE TITLE
Feature request: "ant clean"

### ISSUE THREAD

**Issue by ge0rg** [rtreffer/asmack#1]
It would be nice to have an ant rule to remove all auto-generated files, which is typically called with "ant clean".

**Comment by rtreffer** [comment]
Right, but the problem is it would be asymmetric, ant clean / ant won't really build the system. I'll see how I can change the build system to make the build.xml the entry point
